src/WebJobs.Script/Description/CSharp/CSharpFunctionInvoker.cs

```diff
@@ -31,6 +32,8 @@ public class CSharpFunctionInvoker : ScriptFunctionInvokerBase
         private readonly Collection<FunctionBinding> _inputBindings;
         private readonly Collection<FunctionBinding> _outputBindings;
         private readonly IFunctionEntryPointResolver _functionEntryPointResolver;
+        private readonly FunctionMetadata _functionMetadata;
```

Comment:
When we discussed moving some of the common stuff (file watcher, etc.) from the invokers to the base. The ctor takes the metadata and passes it to the base type.